packages/sampleCommons/config/cbds/landingPage.json

```diff
@@ -26,6 +18,23 @@
               "linkType": "portal",
               "text": "Explore Gen3 Data"
             }
+          },
```

Comment:
Do we wanna just checkout our old SampleCommons/config/cbds? Unless you think Craig messed with it at all (prolly didn't)